engine/apps/slack/models/slack_message.py

```diff
@@ -217,32 +218,3 @@ def send_slack_notification(self, user, alert_group, notification_policy):
                 pass
             else:
                 raise e
-
-    @classmethod
-    def get_alert_group_from_slack_message_payload(cls, slack_team_identity, payload):
```

Comment:
Moved to [`AlertGroupActionsMixin.get_alert_group`](https://github.com/grafana/oncall/pull/2023/files#diff-03c6d717d39ffaa39fd9a2b825e349ccefdaec25fa28296071012b5805e6be2dR18)